gadget/install/install.go

```diff
@@ -94,11 +90,11 @@ func Run(gadgetRoot, device string, options Options, observer gadget.ContentObse
 	// at this point we removed any existing partition, nuke any
 	// of the existing sealed key files placed outside of the
 	// encrypted partitions (LP: #1879338)
-	if options.KeyFile != "" {
-		if err := os.Remove(options.KeyFile); err != nil && !os.IsNotExist(err) {
-			return fmt.Errorf("cannot cleanup obsolete key file: %v", options.KeyFile)
+	sealedKeyFiles, _ := filepath.Glob(filepath.Join(boot.InitramfsEncryptionKeyDir, "*.sealed-key"))
+	for _, keyFile := range sealedKeyFiles {
```

Comment:
BTW is there anything else we want to preserve in that directory if we are reinstalling/installing from scratch?

Reply:
With the current directory layout no, but if we ever set `InitramfsEncryptionKeyDir` to a different location we may need to keep some files in place.